src/circuits/test_circuit/mod.rs

```diff
@@ -38,11 +39,16 @@ use crate::foreign::wasm_input_helper::circuits::WasmInputHelperTableConfig;
 use crate::foreign::wasm_input_helper::circuits::WASM_INPUT_FOREIGN_TABLE_KEY;
 use crate::foreign::ForeignTableConfig;
 
+use super::config::zkwasm_k;
 use super::config::CircuitConfigure;
 use super::image_table::ImageTableConfig;
 
 pub const VAR_COLUMNS: usize = 52;
 
+// Reserve a few rows to keep usable rows away from blind rows.
+// The maximal step size of all tables is bit_table::STEP_SIZE.
+const RESERVE_ROWS: usize = crate::circuits::bit_table::STEP_SIZE;
```

Comment:
We can add check in build.rs to guarantee that bit_table step size is the max step size.